rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -1837,6 +1838,10 @@ public Integer voteFeePerKbChange(Transaction tx, Coin feePerKb) {
             return FEE_PER_KB_GENERIC_ERROR_CODE;
         }
 
+        if(!feePerKb.isPositive()){
```

Comment:
Shouldn't this be programmed as a network upgrade?

Reply:
Guido checked on mainnet and testnet and this function was never called with a negative number, this is not a public function. So we are ok going with this without a network upgrade